deps/wazuh_testing/wazuh_testing/agent.py

```diff
@@ -10,7 +10,7 @@
 
 from wazuh_testing.fim import change_internal_options
 from wazuh_testing.tools import LOG_FILE_PATH, WAZUH_PATH
-from wazuh_testing.tools.monitoring import FileMonitor
+import wazuh_testing.tools.monitoring as monitoring
```

Comment:
```suggestion
from wazuh_testing.tools import monitoring
```

Reply:
Done in 8e0d101bfe9a9fd52bc6a554991e598bc9e00b6b